pkg/crc/cluster/clusteroperator.go

```diff
@@ -50,6 +51,23 @@ func (status *Status) String() string {
 	return "Operators are not ready yet"
 }
 
+func joinWithLimit(names []string) string {
```

Comment:
I'd pass the limit as an argument, but not really important :)

Reply:
done